homeassistant/components/smarttub/entity.py

```diff
@@ -0,0 +1,66 @@
+"""SmartTub integration."""
+import logging
+
+import smarttub
+
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+from homeassistant.util import slugify
+
+from .const import DOMAIN
+from .helpers import get_spa_name
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["climate"]
+
+
+class SmartTubEntity(CoordinatorEntity):
+    """Base class for SmartTub entities."""
+
+    def __init__(
+        self, coordinator: DataUpdateCoordinator, spa: smarttub.Spa, entity_name
+    ):
+        """Initialize the entity.
+
+        Given a spa id and a short name for the entity, we provide basic device
+        info, name, unique id, etc. for all derived entities.
+        """
+
+        super().__init__(coordinator)
+        self.spa = spa
+        self._entity_name = entity_name
+
+    @property
+    def device_info(self) -> str:
+        """Return device info."""
+        return {"identifiers": {(DOMAIN, self.spa.id)}}
```

Comment:
Looks like there are `brand` and `model` properties that can be included here

https://developers.home-assistant.io/docs/device_registry_index#device-properties

Reply:
OK, added